projects/ngx-charts-on-fhir/src/lib/fhir-data/fhir-data.service.ts

```diff
@@ -71,6 +71,83 @@ export class FhirDataService {
       return teardownLogic;
     });
   }
+  addPatientData(reportBPValue: any) {
+    const resource = {
```

Comment:
Can we separate the concerns of  1) creating the Resource object, and 2) making the post request, so that we can reuse the `addPatientData()` function for any kind of data, not just BP?

Reply:
Yes, I will add one more separate function(createResourceData) for creating the Resource Object.